### ISSUE TITLE
Easier installation & setup (e.g., .deb and .rpm packages, portableapps, self-contained zip, etc.)

### ISSUE THREAD

**Issue by gabrielmongefranco** [danielquinn/paperless#426]
Greetings!

     First of all, I want to say that Paperless is a fantastic app! It does one thing and does it *really* well: make a small office truly paperless! Thank you for making this great software suite. I tried several other Document Management Systems and Paperless Office Systems, including Open KM, Mayan EDMS, and Open Prodoc, PaperPort Deluxe, but all were either too complicated or too limited. Paperless falls right in the middle: easy and functional.

      Where Paperless falls flat, however, is in the installation. The setup procedure for bare metal is very complex, even for technical users. The Docker route is fine for Dev Ops and software developers, but definitely not for the target audience (SOHO users) who would have to learn complex Linux commands plus the Docker ecosystem.

      If I may make a suggestion, it would be great if Paperless could be installed in a far easier manner. Ideally, providing .deb and .rpm packages for the major distributions would be best (Debian, Ubuntu, RedHat, Fedora, CentOS, Slackware). I know that's quite an undertaking, so at least creating a Debian and Ubuntu-compatible .deb would open the doors to thousands of SOHO users, since Debian-based distributions tend to be quite popular.

     Another option would be to consider something like a PortableApp or a self-contained .zip. I'm not sure how Open Prodoc does it, but they contain all dependencies in a single Zip. Perhaps it helps them to be Java-based though, which may not work with the Paperless architecture.

      If anything, at the very least providing an install.sh bash script that works in most distributions would be a good start.

Best regards,

Gabriel Mongefranco
Software Developer
[https://gabriel.mongefranco.com](https://gabriel.mongefranco.com)

**Comment by danielquinn** [comment]
All of that sounds really neat.  I would love to have installation be as easy as `pacman -S paperless` or `apt install paperless` or even `emerge paperless` -- but I'm not going to do that.  If you're interested in writing a package yourself, that's awesome and after some testing, I'll merge it.

I just don't have the energy/time/interest in doing this myself is all.

**Comment by knireis** [comment]
I agree with topic starter. The software looks really nice but it is unfortunately too complex for soho user.
I started the Docker way on ubuntu server, but after installing docker (step1) en docker-compose (step2) i'm stuck at step3. Where do i copy those files? Could you please write a more detailed setup instruction?

**Comment by tooomm** [comment]
I feel you @knireis. I plan on using this promising app, but am also already afraid of getting it installed and setup properly...
